eclair-core/src/main/scala/fr/acinq/eclair/payment/send/PaymentLifecycle.scala

```diff
@@ -305,7 +305,7 @@ class PaymentLifecycle(nodeParams: NodeParams, cfg: SendPaymentConfig, router: A
     val duration = now - start
     if (cfg.recordPathFindingMetrics) {
       val fees = result match {
-        case Right(paymentSent: PaymentSent) => paymentSent.feesPaid
+        case Right(paymentSent: PaymentSent) => paymentSent.feesPaid + paymentSent.localFees
         case Left(_) => request match {
           case s: SendPaymentToNode => s.routeParams.getMaxFee(cfg.recipientAmount)
           case _: SendPaymentToRoute => 0 msat
```

Comment:
What is the rationale for setting `0 msat` here?

Reply:
It's actually not possible to reach this case (because for `SendPaymentToRoute`, `recordPathFindingMetrics` is set to false) but the compiler doesn't know it.